src/hocon_token.erl

```diff
@@ -201,9 +201,14 @@ scan_error(Line, ErrorInfo, Ctx) ->
 parse_error(Line, ErrorInfo, Ctx) ->
     throw({parse_error, format_error(Line, ErrorInfo, Ctx)}).
 
-format_error(Line, ErrorInfo, Ctx) ->
+format_error(Line, ErrorInfo, #{filename := F}) ->
     binary_to_list(
         iolist_to_binary(
             [ErrorInfo,
              io_lib:format(" in line ~w. file: ~p",
-                           [Line, hd(hocon_util:get_stack(filename, Ctx))])])).
+                           [Line, hd(F)])]));
+format_error(Line, ErrorInfo, _Ctx) ->
```

Comment:
is this to make the `binary` API work ?
maybe it's easier if `hocon:binary/1` creates a `'nofile'` to make it easier.

Reply:
fix